static/bidder-info/adot.yaml

```
@@ -1,5 +1,6 @@
 maintainer:
   email: "admin@we-are-adot.com"
+  gvlVendorID: 272
```

```suggestion
gvlVendorID: 272
```
J'ai regardé dans les autres yaml ils ont la variable à la racine du yaml, je sais pas si c'est volontaire de l'avoir indenté

---

Bien vu, effectivement c'est à la racine